Remotion/Web/Web.Development.WebTesting.IntegrationTests.RequireUI/App.config

```diff
@@ -0,0 +1,70 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<configuration>
```

Comment:
We need to add this new project to `Project.props` for it to run on CI. We also need to configure it so that it does not run in a docker images as that would prevent it from going non-headless. We should be able to use the configurations define at the top of `Project.props` to do this, although I am not sure how it works exactly :D

Reply:
Yeah I'll definitely need help with that.